src/js/components/Select/Select.js

```diff
@@ -138,7 +142,15 @@ class Select extends Component {
               flex={false}
               style={{ minWidth: 'auto' }}
             >
-              <SelectIcon color='brand' size={size} />
+              <ThemeContext.Extend
+                value={{ color: colorIsDark(theme.select.backgroundColor) ? (
+                  theme.select.icons.color.dark || colorForName('light-4', theme)
+                ) : (
+                  theme.select.icons.color.light || colorForName('brand', theme)
+                ) }}
+              >
+                <SelectIcon size={size} />
```

Comment:
Can we just pass via a color property to `SelectIcon`, rather than extending the icon theme?

Reply:
I can't, take the following example for a custom theme:

```
  const theme = {
    select: {
       icons: {
         color: {
           light: '#ffffff'
         }
       }
    },
  }
```

then if I pass the color as a prop we would have:

`<SelectIcon color="#ffffff" />`

this fails since `#ffffff` is not a color name.

Reply:
I thought we'd just pass the color value straight through?

It also might be handy to use color names in theme objects, to avoid having to create extra variables or copy/paste.

Reply:
I've created https://github.com/grommet/grommet-icons/pull/54 to update grommet-icons to allow more variety in the color property.